### DISCUSSION TITLE
Autoscroll

### DISCUSSION THREAD

**Discussion by avg-guy** [asmyshlyaev177/react-horizontal-scrolling-menu#222] [Q&A]
Hello,
is it possible to implement an autoscroll feature, scrolling to the next item on a setInterval timer?
Thank you!

**Comment by asmyshlyaev177** [comment]
Hi, i think so, you can look into apiRef example https://codesandbox.io/s/react-horizontal-scrolling-menu-v2-apiref-vdr0d?file=/src/index.tsx:855-931

Instead of `apiRef.current.scrollToItem(` you can do `apiRef.current.scrollNext()` and put it in setInterval. Let me know if it works.
